lib/error-reporter.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  reportError: (message, context) => {
+    if (!window || typeof CustomEvent === 'undefined') return
+
+    window.dispatchEvent(new CustomEvent('RecartIntegrationError', {
+      detail: {
+        integration: 'nacelle',
+        message,
+        context
+      }
+    }))
+  }
```

Comment:
[See this related PR](https://github.com/ghostmonitor/gm-tracking-js/pull/1126) for more details